formats/developers/ome-units.txt

```diff
@@ -0,0 +1,253 @@
+The OME System of Units
+=======================
+
+.. topic:: Overview
+
+   This sections explains the OME system of Units.
+
+Implementation
+^^^^^^^^^^^^^^
+
+The `ome.xsd` schema file defines new type enumerations: UnitsLength,
+UnitsTime, UnitsPressure, UnitsAngle, UnitsTemperature,
+UnitsElectricPotential, UnitsPower and UnitsFrequency. These types are used
+throughout the model. Each new unit attribute is in a pair with an existing
+attribute, e.g. PinholeSize and PinholeSizeUnit. The Xsd-Fu application then
```

Comment:
Yes it is correct. 

This is not about the API, this is about the ome system of units and how we use it.